<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OperationDialog</class>
 <widget class="QDialog" name="OperationDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>167</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>621</width>
    <height>167</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>621</width>
    <height>167</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Directory structure scan</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="labelScanFS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tree scan:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="labelScanFileNumberValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelScanFSValue">
         <property name="text">
          <string notr="true">c:</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::NoTextInteraction</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="dirSelectButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelScanDir">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>13</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Traversing:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="labelScanDirValue">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>450</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelScanFileNumber">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>13</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Files read:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>319</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="format">
      <string notr="true">%p%</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>319</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>131</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="okButton">
           <property name="text">
            <string>&amp;OK</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/DataAnalyzer.qrc">
             <normaloff>:/images/tasto-7-architetto-franc-01-300px.png</normaloff>:/images/tasto-7-architetto-franc-01-300px.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="startButton">
           <property name="text">
            <string>&amp;Run</string>
           </property>
           <property name="icon">
            <iconset resource="../../resource/DataAnalyzer.qrc">
             <normaloff>:/images/media-playback-start-300px.png</normaloff>:/images/media-playback-start-300px.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../resource/DataAnalyzer.qrc">
         <normaloff>:/images/tasto-8-architetto-franc-01-300px.png</normaloff>:/images/tasto-8-architetto-franc-01-300px.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resource/DataAnalyzer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>OperationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>OperationDialog</receiver>
   <slot>onCancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirSelectButton</sender>
   <signal>clicked()</signal>
   <receiver>OperationDialog</receiver>
   <slot>onDirSelect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>OperationDialog</receiver>
   <slot>onStart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OperationDialog</sender>
   <signal>progressChanged(int)</signal>
   <receiver>OperationDialog</receiver>
   <slot>onUpdateProgress(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OperationDialog</sender>
   <signal>resultReady()</signal>
   <receiver>OperationDialog</receiver>
   <slot>onResultReady()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OperationDialog</sender>
   <signal>scannedDirChanged(QString)</signal>
   <receiver>OperationDialog</receiver>
   <slot>onScanDirChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OperationDialog</sender>
   <signal>filesRead(unsigned)</signal>
   <receiver>OperationDialog</receiver>
   <slot>onFilesRead(unsigned)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>progressChanged(int)</signal>
  <signal>resultReady()</signal>
  <signal>scannedDirChanged(QString)</signal>
  <signal>filesRead(unsigned)</signal>
  <slot>onDirSelect()</slot>
  <slot>onStart()</slot>
  <slot>onUpdateProgress(int)</slot>
  <slot>onCancel()</slot>
  <slot>onResultReady()</slot>
  <slot>onScanDirChange(QString)</slot>
  <slot>onFilesRead(unsigned)</slot>
 </slots>
</ui>
